packages/addon-dev/src/rollup-hbs-plugin.ts

```diff
@@ -4,19 +4,48 @@ import { readFileSync } from 'fs';
 const backtick = '`';
 
 export default function rollupHbsPlugin(): Plugin {
-  const filter = createFilter('**/*.hbs');
+  const templateFilter = createFilter('**/*.hbs');
+  const moduleFilter = createFilter('**/*.hbs.js');
 
   return {
     name: 'rollup-hbs-plugin',
+    async resolveId(source: string, importer: string | undefined, options) {
+      const resolution = await this.resolve(source, importer, {
+        skipSelf: true,
+        ...options,
+      });
+
+      const id = resolution?.id;
+
+      if (!templateFilter(id)) return null;
```

Comment:
I think we can move this up to be first (no need to `this.resolve` if we are just going to return null)

Reply:
We can't move it up because we don't have an `id` yet.

Reply:
Ah! Thank you!